docs/docs/behaviors/bluetooth.md

```diff
@@ -28,12 +33,14 @@ This will allow you to reference the actions defined in this header such as `BT_
 
 Here is a table describing the command for each define:
 
-| Define   | Action                                                                                                                                                    |
-| -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `BT_CLR` | Clear bond information between the keyboard and host for the selected profile.                                                                            |
-| `BT_NXT` | Switch to the next profile, cycling through to the first one when the end is reached.                                                                     |
-| `BT_PRV` | Switch to the previous profile, cycling through to the last one when the beginning is reached.                                                            |
-| `BT_SEL` | Select the 0-indexed profile by number. Please note: this definition must include a number as an argument in the keymap to work correctly. eg. `BT_SEL 0` |
+| Define          | Action                                                                                                                                                    |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `BT_CLR`        | Clear bond information between the keyboard and host for the selected profile.                                                                            |
+| `BT_NXT`        | Switch to the next profile, cycling through to the first one when the end is reached.                                                                     |
+| `BT_PRV`        | Switch to the previous profile, cycling through to the last one when the beginning is reached.                                                            |
+| `BT_SEL`        | Select the 0-indexed profile by number. Please note: this definition must include a number as an argument in the keymap to work correctly. eg. `BT_SEL 0` |
+| `BT_DISCONNECT` | Disconnect from the 0-indexed profile by number, if it's currently connected and inactive. Please note: this definition must include a number as an       |
```

Comment:
The length of this define kind of sticks out from the other ones. What do you think about abbreviating it, e.g. `BT_DIS` or `BT_DSC`?

Reply:
I'm torn, because those abbreviations aren't especially transparent; you'd have to check the docs to interpret them. NXT, PRV, and SEL all don't have that problem, they're quite self-explanatory.

Reply:
I honestly think `BT_DISC` is descriptive enough, at least as descriptive as `BT_SEL` is, and as long as it's clearly documented, should be fine.

Reply:
Okay, `BT_DISC` works for me, I'll make the change